lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -91,6 +94,69 @@ def test_args():
             "Status: total "
         ), f"Unexpected stdout: '{result.stdout}', stderr: '{result.stderr}'"
 
+    @staticmethod
+    @responses.activate
+    def test_metadata_args():
+        """Test metadata with irregular option/value pair"""
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(
+            main,
+            args=[
+                TestMoveResults.CTRL_SWITCH,
+                TestMoveResults.CTRL_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
+                TestMoveResults.DELY_SWITCH,
+                TestMoveResults.XZST_SWITCH,
+                TestMoveResults.SWSR_SWITCH,
+                TestMoveResults.SWSR_TEXT,
+                TestMoveResults.ACCESS_SWITCH,
+                TestMoveResults.ACCESS_TEXT,
+                TestMoveResults.META_SWITCH,
+                TestMoveResults.META_TEXT_TEST,
+                TestMoveResults.META_TEXT_FOO,
+            ],
+        )
+        assert (
+            result.exit_code == 2
+        ), f"Expected exit code of 2: exit_code = {result.exit_code:d}, stderr: {result.stderr}, stdout: {result.stdout}"
+        assert result.stderr, f"Usage: '{result.stderr}', stdout: '{result.stdout}'"
+        assert "Error: Got unexpected extra argument (pbench.sat:FOO)" in result.stderr
```

Comment:
No, I was not planning to remove l.123

Reply:
OK, then perhaps you might want to tweak it some:  if the assertion fails, then `result.stderr` is probably an empty string, so the output will be
```
Usage: '', stdout: '<...stuff...>'
```
which doesn't really convey much to me in terms of indicating why the test failed.

On the other hand, if you simply remove line 123, then, in the situation where it would have failed, we'll see the failure of the assertion at line 124, in which PyTest will helpfully display the fact that `result.stderr` is empty (not to mention the fact that the assertion expression in line 124 is pretty expressive -- i.e., it shows what we expected, and PyTest will show what we actually got).